ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -228,6 +227,7 @@ export const Homepage = withUserProfile()(class extends React.Component<
               accessTasksLoaded: true
             });
           } else {
+            await this.syncCompliance();
```

Comment:
This isn't in the right place - this `else` clause is effectively dead code; `enforceRegistered` is a server side flag which we are in the process of removing, and should always be `true` at this point. This should be running as a function of the profile state - possibly using a check similar to L215.

Also we'd probably want `syncCompliance()` to complete before setting `accessTasksLoaded`